<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QComboBox" name="deviceComboBox"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Basic information</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <widget class="QLabel" name="as">
                   <property name="text">
                    <string>guid:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>description:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>driver module:</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <widget class="QLabel" name="guidLabel">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="descriptionLabel">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="driverModuleLabel">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="speakerConfigurationGroupBox">
            <property name="title">
             <string>Speaker configurations</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QRadioButton" name="stereo5Subwoofer1Radio">
                 <property name="text">
                  <string>（家庭影院配置）5个环绕扬声器，1个低音炮</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="directPlayRadio">
                 <property name="text">
                  <string>（未配置）直接播放</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="headphoneRadio">
                 <property name="text">
                  <string>耳机</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="monoSpeakerRadio">
                 <property name="text">
                  <string>单个扬声器</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="quadraSpeakerRadio">
                 <property name="text">
                  <string>4声道扬声器</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="stereoSpeakerRadio">
                 <property name="text">
                  <string>双声道（即立体声）扬声器</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Fixed</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>30</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout">
                   <item>
                    <widget class="QRadioButton" name="stereoDegree20Radio">
                     <property name="text">
                      <string>20度弧形摆放</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QRadioButton" name="stereoDegree10Radio">
                     <property name="text">
                      <string>10度弧形摆放</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QRadioButton" name="stereoDegree5Radio">
                     <property name="text">
                      <string>5度弧形摆放</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QRadioButton" name="stereoDegree180Radio">
                     <property name="text">
                      <string>180度弧形摆放</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QRadioButton" name="surroundpeakerRadio">
                 <property name="text">
                  <string>环绕扬声器</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="stereo7Subwoofer1Radio">
                 <property name="text">
                  <string>（[宽、环绕]家庭影院配置）7个环绕扬声器，1个低音炮</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Device capabilities</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QCheckBox" name="certifiedCheckBox">
                 <property name="text">
                  <string>Certified by microsoft</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="rangeBasedCheckBox">
                 <property name="text">
                  <string>Range-based sample rate</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="enumeratedCheckBox">
                 <property name="text">
                  <string>Enumerated driver</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="primary16CheckBox">
                 <property name="text">
                  <string>16bits primary buffer</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="primary8CheckBox">
                 <property name="text">
                  <string>8bits primary buffer</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="monoPrimaryCheckBox">
                 <property name="text">
                  <string>mono primary buffer</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="stereoPrimaryCheckBox">
                 <property name="text">
                  <string>stereo primary buffer</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="secondary16CheckBox">
                 <property name="text">
                  <string>16bits secondary buffer</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="secondary8CheckBox">
                 <property name="text">
                  <string>8bits secondary buffer</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="monoSecondaryCheckBox">
                 <property name="text">
                  <string>mono secondary buffer</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="stereoSecondaryCheckBox">
                 <property name="text">
                  <string>stereo secondary buffer</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>min-max sample rates:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>max mixed buffesr in hardware:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>max static buffers:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>max streaming buffers:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>unallocated buffers:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>unallocated static buffers:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_10">
                   <property name="text">
                    <string>unallocated streaming buffers:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_18">
                   <property name="text">
                    <string>max 3D buffers:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_19">
                   <property name="text">
                    <string>max static 3D buffers:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_20">
                   <property name="text">
                    <string>max streaming 3D buffers:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_21">
                   <property name="text">
                    <string>unallocated 3D buffers:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_22">
                   <property name="text">
                    <string>unallocated static 3D buffers:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_23">
                   <property name="text">
                    <string>unallocated streaming 3D buffers:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_24">
                   <property name="text">
                    <string>total memory of static buffers on sound cards:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_25">
                   <property name="text">
                    <string>free memory of static buffers on sound cards:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_26">
                   <property name="text">
                    <string>largest contiguous block of free memory on sound cards:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_27">
                   <property name="text">
                    <string>transfer rate to static sound buffers:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_28">
                   <property name="text">
                    <string>overhead of mixing software buffers:</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <item>
                  <widget class="QLabel" name="minMaxSampleRateLabel">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="maxHardwareBufferLabel">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="maxStaticLabel">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="maxStreamingLabel">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="unallocatedHardwareBufferLabel">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="unallocatedStaticLabel">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="unallocatedStreamingLabel">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="max3DLabel">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="maxStatic3DLabel">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="max3DStreamingLabel">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="unallocated3DLabel">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="unallocatedStatic3DLabel">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="unallocatedStream3DLabel">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="totalMemoryLabel">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="freeMemoryLabel">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="largestContinugousLabel">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="transferRateLabel">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="overheadLabel">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
